kitchen.validate.yml

```diff
@@ -53,7 +42,19 @@ suites:
         - cron_disabled_selected_daily_and_weekly_jobs
         - openssh_installed
         - ami_cleanup_file_created
-        # TODO: enable following controls when testing install.rb recipe
-        # - efa_conflicting_packages_removed
-        # - efa_prereq_packages_installed
-        # - efa_installed
+        - c_states_kernel_configuration
+        - chrony_installed_and_configured
+        - clusterstatusmgtd_files_created
+        - disable_selinux
+        - efa_conflicting_packages_removed
+        - efa_installed
+        - efa_prereq_packages_installed
+        - efs_utils_installed
+        - gc_thresh_values_configured
+        - license_readme_created
+        - mysql_client_installed
+        - mysql_client_source_node_created
+        - node_attributes
```

Comment:
Nitpick. Could we rename it to past participle?

Reply:
Done